### DISCUSSION TITLE
Request labels, milestones, and assignees support

### DISCUSSION THREAD

**Discussion by micnncim** [community/community#3741] [Discussions]
Currently, Discussions doesn't support labels, milestones, or assignees. I hope it supports them to make clear discussions. Especially, when converting an issue to a discussion, it'd be helpful to keep the tagged labels, milestones, and assignees.

**Comment by evi-liu** [comment]
Hi @micnncim! Good news - we are adding labels to Discussions very soon :). Would you like to help us beta test it when it's available?

As for milestones and assignees, I'd love to hear more about how you would use them in your community. Any specific examples on how they'd be valuable?

**Reply by micnncim** [reply]
Hi @evi-liu 😄 Thank you for your reply!

> Would you like to help us beta test it when it's available?

Glad to hear. I'd like to join beta testing with my company's organization if possible.

As for milestones and assignees, as one of clear examples, we'd like to have a single discussion for all questions, where we'd assign someone suitable for answering to each discussion and put them into a milestone suitable among our milestones to clarify the kind of questions (and label).

**Reply by evi-liu** [reply]
Which repository would you like to enable Discussions labels on?

**Reply by micnncim** [reply]
I'd like to use it in a private repository if possible.

**Reply by dlech** [reply]
> Which repository would you like to enable Discussions labels on?

https://github.com/pybricks/support

**Reply by aeslaughter** [reply]
> As for milestones and assignees, I'd love to hear more about how you would use them in your community. Any specific examples on how they'd be valuable?

I am also interested in the assignees features for [MOOSE](https://github.com/idaholab/moose/discussions). This is a project that contains various  modules, developed by different teams. It would be very useful to be able to assign a discussion to the team lead rather than adding an "answer" with "Hey, @somebody please look at this".

**Reply by jprestop** [reply]
@evi-liu 
> As for milestones and assignees, I'd love to hear more about how you would use them in your community. Any specific examples on how they'd be valuable?

My team is also interested in the assignee feature for [METplus](https://github.com/dtcenter/METplus/discussions).  Our project has multiple components and is developed by different teams as well.  It would be very useful to assign a discussion to the expert for the given Discussions and easier for our team to track Discussions that they are responsible for.
